qiskit/compiler/transpiler.py

```diff
@@ -451,13 +492,83 @@ def _log_transpile_time(start_time, end_time):
     logger.info(log_msg)
 
 
+def _parse_basis_gates(basis_gates, backend, inst_map, skip_target):
+    name_mapping = CUSTOM_MAPPING
+    standard_gates = get_standard_gate_name_mapping()
+    control_flow_gates = {
```

Comment:
There is a static set defined for these names here: https://github.com/Qiskit/qiskit/blob/main/qiskit/circuit/controlflow/__init__.py#L27 Although we shouldn't need to match on "continue" or "break" they only appear inside control flow blocks and we shouldn't ever see them at this level.